packages/flutter/test/material/chip_test.dart

```diff
@@ -188,6 +188,91 @@ void _expectCheckmarkColor(Finder finder, Color color) {
   );
 }
 
+Future<void> pumpFrames({
+  WidgetTester tester,
+  int frames,
+  Duration duration,
+}) async {
+  int i;
+  for (i = 0; i < frames; i ++) {
```

Comment:
```suggestion
  for (int i = 0; i < frames; ++i) {
```
(contains `i` inside the loop, and is very slightly faster, since `++i` doesn't need to return `i`).

Reply:
Fixed.